src/sumoLogger.js

```diff
@@ -45,7 +46,9 @@ class SumoLogger {
                 ? newConfig.returnPromise
                 : true,
             clientUrl: newConfig.clientUrl || '',
+            useIntervalOnly: newConfig.useIntervalOnly || false,
```

Comment:
Worth adding this config option into the README?

Reply:
done